src/Conversions.cc

```diff
@@ -259,16 +259,25 @@ msgs::Material ignition::gazebo::convert(const sdf::Material &_in)
     if (workflow)
     {
       pbrMsg->set_metalness(workflow->Metalness());
-      pbrMsg->set_metalness_map(workflow->MetalnessMap());
+      pbrMsg->set_metalness_map(workflow->MetalnessMap().empty() ? "" :
```

Comment:
Are the empty checks necessary? If so, I'd suggest adding support for that case within `asFullPath`.

Reply:
Yeah, I added the empty checks after initially not having them.